pkg/machinery/config/types/network/default_action_config.go

```diff
@@ -4,6 +4,8 @@
 
 package network
 
+//docgen:jsonschema
```

Comment:
This is a marker to hint docgen to scan this file's structs for the JSON schema (files without this are ignored). Now that we scan all config dirs at once, listing each and every file as command line arg felt tedious, so I went with this way. Open for better ideas though.